docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -47,6 +47,10 @@ for instance:
               </exec>
           </target>
         ```
+- update PHPStan to the 0.11 version ([#826](https://github.com/shopsys/shopsys/pull/826))
+    - change version of phpstan/phpstan to ^0.11 in your composer.json and update dependencies with `composer update`
```

Comment:
what about nikic/php-parser?

Reply:
well, it's not a dependency in project-base. Only in framework, so I don't think it should be mentioned.

Reply:
oh, of course, you are right :slightly_smiling_face: